docs/api.md

```diff
@@ -550,6 +550,18 @@ const logger = pino(pinoOptions)
 * See [`message` log method parameter](#message)
 * See [`logMethod` hook](#logmethod)
 
+<a id="errors"></a>
+#### Errors
+
+Errors can be supplied as either the first parameter or if already using `mergingObject` then as the `err` on the `mergingObject`.
```

Comment:
This is accurate if the user has not customized the serializers, e.g.:

```js
const logger = pino({
  serializers: {
    error: stdSerializers.error
  }
})
```

In the above, the `err` serializer, as described in this documentation, will not be defined. Instead, it is renamed to `error`. So:

```js
// Will not serialize
logger.error({ err: Error('boom') })

// Will serialize
logger.error({ error: Error('boom') })
```

Reply:
Tried to point this out, let me know if you have a more concise sentence.